x/incentives/keeper/gauge.go

```diff
@@ -93,8 +94,20 @@ func (k Keeper) SetGaugeWithRefKey(ctx sdk.Context, gauge *types.Gauge) error {
 	}
 }
 
-// CreateGauge creates a gauge and sends coins to the gauge.
-func (k Keeper) CreateGauge(ctx sdk.Context, isPerpetual bool, owner sdk.AccAddress, coins sdk.Coins, distrTo lockuptypes.QueryCondition, startTime time.Time, numEpochsPaidOver uint64) (uint64, error) {
+// CreateGauge creates a gauge with the given parameters and sends coins to the gauge.
+// There can be 2 kinds of gauges for a given set of parameters:
+// * lockuptypes.ByDuration - a gauge that incentivizes one of the lockable durations.
+// For this gauge, the pool id must be 0. Fails if not.
+//
+// * lockuptypes.NoLock - a gauge that incentivizes pools without locking. Initially,
+// this is meant specifically for the concentrated liquidity pools. As a result,
+// if NoLock gauge is being created, the given pool id must be non-zero, the pool
+// at this id must exist and be of a concentrated liquidity type. Fails if not.
+// Additionally, lockuptypes.Denom must be either an empty string, signifying that
+// this is an external gauge, or be equal to types.NoLockInternalGaugeDenom(poolId).
```

Comment:
for clarity what do you think of this instead of all types.NoLockInternalGaugeDenom(poolId) in comments?  
```suggestion
// this is an external gauge, or be equal to "no-lock/poolId"
```

Reply:
hmm, I prefer the current way but open to changing if more reviewers favor the proposal.

It would be `"no-lock/i/<pool id>"`, specifically, for internal gauges

Reply:
Going to resolve since no strong opinions expressed